src/site/antora/modules/ROOT/partials/manual/systemproperties/properties-context-selector.adoc

```diff
@@ -55,4 +55,17 @@ link:../javadoc/log4j-core/org/apache/logging/log4j/core/osgi/BundleContextSelec
 Creates a separate logger context per OSGi bundle and synchronous loggers
 
 link:../javadoc/log4j-core/org/apache/logging/log4j/core/selector/JndiContextSelector.html[`org.apache.logging.log4j.core.selector.JndiContextSelector`]::
-Creates loggers contexts based on a JNDI lookup and synchronous loggers See xref:jakarta.adoc#jndi-configuration[Web application] for details.
\ No newline at end of file
+Creates loggers contexts based on a JNDI lookup and synchronous loggers See xref:jakarta.adoc#jndi-configuration[Web application] for details.
+
+There are also third-party context selector implementations:
+
+[IMPORTANT]
+====
+These context selectors are provided by **third-party** vendors and are not maintained by the Apache Software Foundation.
```

Comment:
```suggestion
These lookups are provided by **third-party** vendors and are not maintained by the link:{logging-services-url}[Apache Logging Services] (Log4j, Log4cxx, Log4net) project.
```

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/3534/commits/84c6da7b084512d2e9d54b6cb42c8b718a80b8ed